package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "webpack-boilerplate",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "jest",
```

Comment:
- [ ] Please, do some workaround with your `test` config as it is complaining on my end and I cannot run the test. 
![Screenshot from 2021-08-07 00-59-55](https://user-images.githubusercontent.com/64894927/128559397-d8eab1b6-e934-4c5b-9cbd-62d14d6b19f5.png)